src/mscorlib/src/System/Diagnostics/Eventing/EventSource.cs

```diff
@@ -2494,6 +2494,7 @@ internal void DoCommand(EventCommandEventArgs commandArgs)
 #if FEATURE_MANAGED_ETW
             if (m_provider == null)     // If we failed to construct
                 return;
+#endif // FEATURE_MANAGED_ETW
 
             m_outOfBandMessageCount = 0;
             bool shouldReport = (commandArgs.perEventSourceSessionId > 0) && (commandArgs.perEventSourceSessionId <= SessionMask.MAX);
```

Comment:
It looks like there is other ETW specific code (e.g. the call to UpdateEtwSession) that is no longer in the FEATURE_MANAGED_ETW ifdef.  It is however, inside of FEATURE_ACTIVITY_SAMPLING, which I'm assuming is not enabled.  If not enabled, we can probably just file an issue to track it so that David can go back and fix it.